frontends/election-manager/src/components/navigation_screen.tsx

```diff
@@ -26,12 +26,14 @@ interface Props {
   children: React.ReactNode;
   centerChild?: boolean;
   flexColumn?: boolean;
+  flexRow?: boolean;
```

Comment:
While I know we shouldn't allow someone to pass both `flexColumn` and `flexRow` at the same time, I'm 1) trusting that consumers will know not to do that and 2) maintaining parity with the underlying `Main` component in libs/ui: https://github.com/votingworks/vxsuite/blob/8ae4877abec0a14e00dbcbc8e3d5baf567cb7306/libs/ui/src/main.tsx#L6-L7

Reply:
Seems fine. Maybe the `Main` component could have an assertion that both aren't passed.